packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -97,6 +98,11 @@ abstract contract CrossDomainMessenger is
      */
     address public otherMessenger;
 
+    /**
+     * @notice Reserve extra slots (to a total of 50) in the storage layout for future upgrades.
+     */
+    uint256[46] private __gap;
```

Comment:
Why not put this gap underneath `blockedSystemAddresses`?

Reply:
ah, good catch.